arachne-commons/src/main/java/com/odysseusinc/arachne/commons/api/v1/dto/CommonAuthenticationRequest.java

```diff
@@ -57,4 +57,8 @@ public void setPassword(String password) {
 		this.password = password;
 	}
 
+	public void normalizeLogin() {
```

Comment:
If DTO requires email in lower case why there is a separate method? What is the contract? Why and when I should use this method?